carla-spice-sync-service/Validation.cs

```diff
@@ -2,7 +2,7 @@
 using System.Text.RegularExpressions;
 using Gov.Lclb.Cllb.Interfaces;
 using Gov.Lclb.Cllb.Interfaces.Spice.Models;
-using Microsoft.Extensions.Logging;
+using Serilog;
```

Comment:
I like the usage of a better logger in this service and the general code cleanup